requirements.bazel.txt

```diff
@@ -14,3 +14,6 @@ chardet==3.0.4
 certifi==2017.4.17
 idna==2.7
 googleapis-common-protos==1.5.5
+gevent==21.1.2
+zope.event==4.5.0
+setuptools==44.1.1
```

Comment:
Do we need `setuptools` in the Gevent Bazel tests?

Reply:
The `setuptools` pin is fall-out from the `rules_python` upgrade. I found that by default, `rules_python` was injecting a `setuptools` version that caused syntax errors for certain Kokoro jobs. But since I've turned `rules_python` into a stub for them, I guess I can give it another shot. Let's see what Kokoro has to say on the matter.

Reply:
Nope. Still an issue. Putting it back.

```
  File "/root/.cache/bazel/_bazel_root/d2dc70c3d9da3fab488ba0dcbbd35051/sandbox/processwrapper-sandbox/3484/execroot/com_github_grpc_grpc/bazel-out/k8-py2-fastbuild/bin/src/python/grpcio_tests/tests/interop/_secure_intraop_test.python2.runfiles/grpc_python_dependencies/pypi__setuptools/pkg_resources/__init__.py", line 1367
    raise SyntaxError(e) from e
```